search-solr/bitnami/solr/server/solr/search/conf/managed-schema.xml

```diff
@@ -174,6 +174,7 @@
       <tokenizer name="standard"/>
       <filter name="stop" ignoreCase="true" words="lang/stopwords_en.txt"/>
       <filter name="lowercase"/>
+      <filter class="solr.ASCIIFoldingFilterFactory"/>
```

Comment:
hey lekshmi this is the right idea, but you're adding it to the wrong field type here. If you look in the api you can see the solr fields that are used by for names and then you can search those in here and see the field types you need to update.

Can you make sure to add a test to the existing test suite for solr for this too? I will run it before approving because if there are issues because it takes a lot of time to redeploy